app/models/assignment.rb

```diff
@@ -732,7 +733,8 @@ def summary_csv(user)
     headers[0] << 'Bonus/Deductions'
     headers[1] << ''
 
-    maximum_mark = max_mark(:all)
+    result_ids = groupings.pluck('results.id').uniq.compact
+    extra_marks_hash = Result.get_total_extra_marks(result_ids, max_mark: max_mark(:all))
```

Comment:
I just noticed this in the diff, but we should use `max_mark`, not `max_mark(:all)` here. (This was a bug from before, but is good to fix here anyway.)